default.nix

```diff
@@ -2,7 +2,9 @@
 #
 # SPDX-License-Identifier: CC0-1.0
 
-(import (fetchTarball
-  "https://github.com/edolstra/flake-compat/archive/master.tar.gz") {
-    src = builtins.fetchGit ./.;
-  }).defaultNix
+let
+  flake = (import (fetchTarball
+    "https://github.com/edolstra/flake-compat/archive/master.tar.gz") {
+      src = builtins.fetchGit ./.;
+    }).defaultNix;
+in flake // flake.packages.${builtins.currentSystem}
```

Comment:
AFAIU the gist of this change is `// flake.packages.${builtins.currentSystem}`. Why is it needed? How is it related to "Add information about building with Nix"?

Reply:
It makes using pre-flake Nix easier. I really don't want to have `nix-build -A packages.x86_64-linux.backend-server` instead of `nix-build -A backend-server` in the readme :)

Reply:
Ah, ok, thanks.